server/controllers/admin/services.js

```diff
@@ -0,0 +1,225 @@
+    sql =
+      'SELECT s.id, s.name, s.enterprise_id, e.name, e.abbr, cc.id AS cc_id, ' + 
+      'cc.text AS cost_center_name, pc.id AS pc_id, pc.text AS profit_center_name ' +
+      'FROM service AS s JOIN enterprise AS e ON s.enterprise_id = e.id ' +
+      'LEFT JOIN cost_center AS cc ON s.cost_center_id = cc.id LEFT JOIN ' +
+      'profit_center AS pc ON s.profit_center_id = pc.id';
+  }
+
+  sql += ' ORDER BY s.name;';
+
+  db.exec(sql)
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+}
+
+/**
+* Create a service in the database
+*
+* @param {object} req The express request object
+* @param {object} res The express response object
+* @param {object} next The express object to pass the controle to the next middleware
+*
+* @example
+* // POST /services : Insert a service
+* var services = require('finance/services');
+* services.create(req, res, next);
+*/
+
+function create (req, res, next) {
+  'use strict';
+
+  var record = req.body;
+  var createServiceQuery = 'INSERT INTO service SET ?';
+
+  delete record.id;
+
+  db.exec(createServiceQuery, [record])
+    .then(function (result) {
+      res.status(201).json({ id: result.insertId });
+    })
+    .catch(next)
+    .done();
+}
+
+/**
+* Update a service in the database
+*
+* @param {object} req The express request object
+* @param {object} res The express response object
+* @param {object} next The express object to pass the controle to the next middleware
+*
+* @example
+* // PUT /services : update a service
+* var services = require('admin/services');
+* services.update(req, res, next);
+*/
+
+
+function update (req, res, next) {
+  'use strict';
+
+  var queryData = req.body;
+  var serviceId = req.params.id;
+  var updateServiceQuery = 'UPDATE service SET ? WHERE id = ?';
+
+
+  delete queryData.id;
+
+  if(!isValidData(queryData)) {
+    return next(new req.codes.ERR_BAD_VALUE());
+  }
+
+  lookupService(serviceId, req.codes)
+    .then(function () {
+      return db.exec(updateServiceQuery, [queryData, serviceId]);
+    })
+    .then(function (result) {
+      return lookupService(serviceId, req.codes);
+    })
+    .then(function (service) {
+      res.status(200).json(service);
+    })
+    .catch(next)
+    .done();
+}
+
+/**
+* Remove a service in the database
+*
+* @param {object} req The express request object
+* @param {object} res The express response object
+* @param {object} next The express object to pass the controle to the next middleware
+*
+* @example
+* // DELETE /services : delete a service
+* var services = require('admin/services');
+* service.remove(req, res, next);
+*/
+
+function remove (req, res, next) {
+  var serviceId = req.params.id;
+  var removeServiceQuery = 'DELETE FROM service WHERE id=?';
+
+  lookupService(serviceId, req.codes)
+    .then(function () {
+      return db.exec(removeServiceQuery, [serviceId]);
+    })
+    .then(function () {
+      res.status(204).send();
```

Comment:
Just so that you  know (and you don't need to change this code), if all you need to do is send a status, the following two are equivalent:
```js
res.status(204).send();
// ... is the same as ...
res.sendStatus(204);
```
[Relevant ExpressJS documentation](http://expressjs.com/en/4x/api.html#res.sendStatus).